addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -2049,19 +2049,21 @@ var SnippetsMenu = Widget.extend({
         }
 
         // Prepare the functions
-        var functions = {
-            is: function ($from) {
-                return $from.is(selector) && $from.filter(filterFunc).length !== 0;
-            },
-        };
+        var functions = {};
         if (noCheck) {
+            functions.is = function ($from) {
+                return $from.is(selector) && $from.filter(filterFunc).length !== 0;
+            };
             functions.closest = function ($from, parentNode) {
                 return $from.closest(selector, parentNode).filter(filterFunc);
             };
             functions.all = function ($from) {
                 return ($from ? dom.cssFind($from, selector) : $(selector)).filter(filterFunc);
             };
         } else {
+            functions.is = function ($from) {
+                return $from.is(selector) && self.getEditableArea().find($from).length && $from.filter(filterFunc).length !== 0;
```

Comment:
Not sure but are you supposed to return true also if `$from` itself is an editable area? Seems like `closest` do it, no?

Reply:
Yes, you're right :+1: , I made the change